pyflakes/checker.py

```diff
@@ -1728,6 +1728,13 @@ def DICT(self, node):
                         )
         self.handleChildren(node)
 
+    def IF(self, node):
+        if isinstance(node.test, ast.Tuple) and node.test.elts != []:
```

Comment:
I was going to suggest this change to `and node.test.elts` but I see below that `ASSERT` does the same thing